### PR TITLE
Easier shoe silencing

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: FlattestGuitar
tweak: You can now silence shoes just by using some duct tape on them! Shoe rags no longer exist.
/:cl:

Seriously, who needs wirecutters to cut duct tape?

No more shoe rags. Just click  with duct tape on a pair of shoes. You can silence 2.5 pairs of shoes with a single roll.

### CODE REVIEW

**Comment by KasparoVy** [comment]
What the heck?

**Comment by Crazylemon64** [comment]
ehhhh, putting an explicit buff/tool item in the loadout system...? 😕

**Comment by FlattestGuitar** [comment]
a) They are highly illogical in their behavior.

b) They add zero gameplay depth.

c) They don't even have a custom sprite.

d) This makes my roundstart ritual quicker by a solid 6.78 seconds.

@Crazylemon64 Considering the abundance of duct tape, I think it's a reasonable addition. I can remove it if it's really that much of a push, but I'd argue cigarettes give you more of a buff and they're also in the loadout system.

**Comment by Fox-McCloud** [comment]
Yeah, this really shouldn't be a loadout item.

Also, the crafting menu is the future; not more slap crafting.

**Comment by FlattestGuitar** [comment]
It's not slap crafting!

Shoe rags make no sense as an item. You don't craft a coat of paint before applying it to a wall.

**Comment by granodd** [comment]
Footsteps don't make sound 90% of the time anyways. Not unless you're wearing jackboots and/or walking on plating/wood. I suppose this would make sense if you are trying to slip through maintenance unheard, though.

Another thing to consider is what all you can muffle with this. If you can muffle no-slips and/or by some voodoo magbolts, I'd say they'd lose their special quality until you pull off the tape (maybe not advanced ones). As for the materials for it being available in a loadout, I'm sensing some small ulterior motives off of the whole "makes my roundstart ritual quicker by a solid 6.78 seconds". I'd say to keep this kind of thing out of the load out for the same reason you don't have access to other items you need to go fetch after round start.

**Comment by sothangel** [comment]
If this is going to be added, tape shouldn't be in the loadout. Silenced jackboots would become the de facto norm.

**Comment by granodd** [comment]
Not like it really matters. No where in the security officer's SoP does it say they need to wear their boots, and if an officer wants to be silent while moving around they'd just take off the jackboots. Which, in my opinion, is really meta. I can't name the number of times I've been snagged by an officer because their shoes made no sound, usually in addition to the fact that they're dressed nothing like an officer.

But that's more to do with how officers dress themselves and less to do with tape on boots. Putting it in the loadout would make it a much more common thing, though.

**Comment by SamHPurp** [comment]
Tbh - Silenced jackboots need a nerf. There needs to be some trade off for silencing them. Currently there is zero.

**Comment by alexkar598** [comment]
In any cases if its going to be in the load out menu which I don't like please say so in the CL

**Comment by taukausanake** [comment]
Make the tape come off after so many steps. Duct tape is invincible yes, but you're going to be running around a lot. They should come off and require re-taping.

I don't think I'd want to use the crafting menu with two items, at least in the instance that one item is being applied to the other. Way easier to just click one object with the other rather than open a menu and find the thing I want to make.

Duct tape really shouldn't be in the loadout screen just like how all other tools and components aren't in it

**Comment by Anticept** [comment]
@SamHPurp it does kind of have a drawback. The jackboots do have a suppressive effect on people's behavior.

Even if you do put in a drawback anyways, people will just use shoes. The question is, what ADVANTAGE does jackboots give me?

**Comment by FalseIncarnate** [comment]
@Anticept an extra second worth of strip delay to have them stolen by the clown who just slipped you. They don't even have armor.

I think the general consensus is that adding tape to the loadout is an unneeded and unwanted addition. The fact that you didn't include it in the changelog just seems sketchy since you mentioned it in the PR anyways...

**Comment by FlattestGuitar** [comment]
Removed tape from loadout. I still don't understand the logic, but I won't argue against it.

I will not be making any further changes to the functionality of this PR other than tweaking numbers.

File: code/game/objects/items/weapons/tape.dm [hunk 1]
```diff
@@ -40,6 +40,28 @@
 		qdel(src)
 
 
+/obj/item/stack/tape_roll/afterattack(atom/target, mob/user, proximity)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/tape.dm]: Good heavens, we have procs for using tape that makes 99% of this redundant. Heaven's almighty.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -175,28 +175,6 @@
 	item_color = "noble_boot"
 	item_state = "noble_boot"
 
-
-/obj/item/clothing/shoes/attackby(obj/item/I, mob/user, params)
```
Comment (Fox-McCloud) [inline code/modules/clothing/shoes/miscellaneous.dm]:
```
/obj/item/clothing/shoes/attackby(obj/item/I, mob/user, params)
	if(istype(I, /obj/item/stack/tape_roll))
		var/obj/item/stack/tape_roll/TR = I
		if(TR.use(4))
			silence_steps = 1
			shoe_sound = null
	else
		return ..()
```

**Pull request merged**